flow/packet-headers/icmpv6.yaml

```diff
@@ -0,0 +1,43 @@
+components:
+  schemas:
+    Flow.Icmpv6:
+      description: ICMPv6 packet header
+      type: object
+      properties:
+        type:
+          x-field-pattern:
+            x-constants:
+              destination_unreachable: 1
+              packet_too_big: 2
+              time_exceeded: 3
+              parameter_problem: 4
+              echo_reply: 129
+              echo_request: 128
+              multicast_listener_query: 130
+            description: >-
+              ICMPv6 Type
+            format: integer
+            length: 8
+            default: 4
+            features: [count, metric_group]
+        code:
+          x-field-pattern:
+            description: >-
+              ICMPv6 subtype
+            format: integer
+            length: 8
+            default: 0
+            features: [count, metric_group]
+        checksum:
+          x-field-pattern:
+            description: >-
+              Internet checksum
+            format: integer
+            length: 16
+        message_body:
```

Comment:
missing default

Reply:
For Checksum any default value is not added.  I verified with the other protocols as we..

Reply:
the default is missing from the message_body

Reply:
Added default